pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -147,8 +163,35 @@ func (c *Controller) logPacket(pktIn *ofctrl.PacketIn) error {
 		return fmt.Errorf("received error while handling packetin for NetworkPolicy: %v", err)
 	}
 
-	// Store log file
-	AntreaPolicyLogger.Printf("%s %s %s %s SRC: %s DEST: %s %d %s", ob.tableName, ob.npRef, ob.disposition, ob.ofPriority, ob.srcIP, ob.destIP, ob.pktLength, ob.protocolStr)
+	// Create 1 sec buffers for deduplicating non-Allow packet log.
```

Comment:
it would be nice to update the comment at L149 for this func to include how logging is done including how thread safe is the updates to the map